packages/reports/addon/services/navi-visualizations.js

```diff
@@ -48,7 +48,7 @@ export default Ember.Service.extend({
    */
   all() {
     // Find all visualizations registered in requirejs under the namespace "components/navi-visualizations"
-    let visualizationRegExp = new RegExp(`^(?:${config.modulePrefix}/)?navi-visualizations/manifests/([a-z-]*)$`),
+    let visualizationRegExp = new RegExp(`^(?:${config.modulePrefix}/)?manifests/([a-z-]*)$`),
```

Comment:
Are you sure we want to remove this? Aren't the visualizations still in `.../components/navi-visualizations/...`?

Reply:
The manifests are not.